engine/config/routes.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 GoodJob::Engine.routes.draw do
-  root to: 'executions#index'
+  root to: redirect(path: 'jobs')
```

Comment:
Since there is talk of eventually adding a different home page for the dashboard, I made this a redirect to the jobs tab so we don't later break any links pointing to specific filters.